### DISCUSSION TITLE
Burger Menu

### DISCUSSION THREAD

**Discussion by tikovolpe** [tailwindlabs/tailwindcss#11932] [Help]
Hey guys!

I'm a beginner in tailwindcss and I'm having a hard time here in making a burger menu work because I'm also starting in js.

I did a search here in the repository but to no avail.

My code is like this;

<div>
                <button class="hamburguer block text-white text-2xl lg:hidden">
                    
                </button>
            </div>
            <div class="hidden lg:block px-4 text-lg mt-10 text-white">
                <nav>
                    <ul class="hidden lg:flex">
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Mensagem do CEO</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">DESTAQUES 2022</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Reconhecimento</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Materialidade</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">A Brisanet</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Governança</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Desempenho econômico</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg border-r px-5 border-white hover:text-azul-brisanet"><a href="">Social</a></li>
                      <li class="item-nav-inf lg:text-sm xl:text-lg px-5 border-white hover:text-azul-brisanet"><a href="">Meio Ambiente</a></li>
                    </ul>
                  </nav>
            </div>

What can you guide me on how to make this burger menu open occupying the entire screen of the mobile device?

Thanks

**Comment by wongjn** [comment]
You'd want a button that has a [`click`](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event) [event listener](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events). Inside this listener, you could change some of the classes attached to the menu to display or hide it. To have the menu full screen, you could consider having it [`position: fixed`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#fixed) and [`inset: 0`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) to cover the full viewport.

[JSFiddle example](https://jsfiddle.net/672zhd0x/)
